bbb_presentation_video/renderer/tldraw/shape/text.py

```diff
@@ -95,18 +94,22 @@ def finalize_label(
     layout = Pango.Layout(pctx)
     layout.set_auto_dir(True)
     layout.set_font_description(font)
-    if shape.size.width > 0:
-        layout.set_width(int(shape.size.width * Pango.SCALE))
-    if shape.size.height > 0:
-        layout.set_height(int(shape.size.height * Pango.SCALE))
-    layout.set_wrap(Pango.WrapMode.WORD_CHAR)
+    layout.set_line_spacing(0.4)
     layout.set_alignment(Pango.Alignment.CENTER)
 
     layout.set_text(shape.label, -1)
 
-    (_, layout_height) = layout.get_pixel_size()
-    height_offset = (shape.size.height - layout_height) / 2
-    ctx.translate(0, height_offset)
+    (layout_width, layout_height) = layout.get_pixel_size()
+    if style.dash is DashStyle.DRAW or isinstance(shape, TriangleShape):
+        width_offset = (shape.size.width - layout_width) / 2 
```

Comment:
Rather than using a fixed division by 2, we should be using the `labelPoint` values from the shape: https://github.com/bigbluebutton/bbb-presentation-video/blob/ade8d9d6fbae4a80d5c872888c0cca0105bc1033/bbb_presentation_video/renderer/tldraw/shape/__init__.py#L67
which contains a value from 0 to 1 indicating the position of the label relative to the shape horizontally and vertically.